### DISCUSSION TITLE
Using calibre to generate wordwise for books in multiple languages is not working

### DISCUSSION THREAD

**Discussion by Cecilita17** [xxyzz/WordDumb#248] [Q&A]
Hello! I'm having trouble getting calibre to add word wise to 2 books in different languages. The first book is in dutch, it gets added to my kindle and works fine, but when I add the Russian book, the duch one gets bugged and the wordwise and x-ray take it as if it's Russian words

Sorry if I'm not being clear, it's weird because I don't touch anything about the dutch book, however the translations get all messed up with incoherent words

I've tried removing the dutch book from calibre but the one that's already uploaded to my kindle seems to get affected when I change the wordwise settings before making the Russian book 

When I create the books using the command, only epub works. I've tried mobie and azw3 and they don't get wordwise added at all 

Any ideas what I can do? 

Also, why even if I enable all words, there are still words that don't get translated? 

Thank you very much for such useful contribution!

**Comment by xxyzz** [comment]
All Wiktionary word wise gloss database files overwrite the same file on Kindle, you can only use one language(book or gloss) at a time.

The command line tool can't send files to Kindle.

Please try the test version build from master branch, it can be downloaded from GitHub Actions, the test version can find all enabled words.

**Reply by Cecilita17** [reply]
Ohh I understand, one language for calibre 

I'm gonna be trying the test version today, thank you :) 

Regarding the command line tool, is there a way to make it create azw3 that actually have wordwise in it? Even if it can't send it to the kindle. Or is it only for epub? 

Thank you for the quick answers!

**Reply by xxyzz** [reply]
This plugin creates word wise file(book file not changed) in the same folder of the book file, use it in GUI mode to send all files to Kindle. Most users shouldn't use the command line tool.
